set-web/src/main/java/eu/europeana/set/web/service/controller/jsonld/WebUserSetRest.java

```diff
@@ -152,34 +145,31 @@
 	@ApiOperation(notes = SwaggerConstants.SEARCH_HELP_NOTE, value = "Retrieve a user set", 
 				nickname = "retrieve", response = java.lang.Void.class)
 	public ResponseEntity<String> getUserSet(
-			@RequestParam(value = WebUserSetFields.PARAM_WSKEY, required = false) String wskey,
 			@PathVariable(value = WebUserSetFields.PATH_PARAM_SET_ID) String identifier,
-			@RequestParam(value = WebUserSetFields.USER_TOKEN, required = false, defaultValue = WebUserSetFields.USER_ANONYMOUNS) String userToken,			
 			@RequestParam(value = WebUserSetFields.PROFILE, required = false, defaultValue = WebUserSetFields.PROFILE_MINIMAL) String profile,			
 			HttpServletRequest request) throws HttpException {
 
 		String action = "get:/set/{identifier}{.jsonld}";
-		return getUserSet(wskey, profile, identifier, request, action);
+		return getUserSet(profile, identifier, request, action);
```

Comment:
the verify read access method must be invoked here, the wskey param is still available in the read methods

Reply:
verifyReadAccess invoked as suggested